src/App.js

```diff
@@ -1,13 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import Header from './Header';
 import Todo from './Todo';
 
 function App() {
   return (
     <>
-    <Todo></Todo>
+      <Header></Header>
+      <Todo></Todo>
```

Comment:
what Todo ? add todo? todoApp? todoList of what? @Kunal-inno make sure the take proper naming conventions for all things